README.md

```diff
@@ -1,13 +1,6 @@
 ## Komoju for WooCommerce.
 
-Japanese payments for WooCommerce.
-This plugin supports the following Japanese payment methods:
-
-* Credit Card (クレジットカード)
-* Convenience Store (コンビニ)
-* PayEasy (ペイジー)
-* WebMoney (ウェブマネー)
-* Bank Transfer (銀行振込)
+Provides access to Komoju payment methods through the WooCommerce checkout.
```

Comment:
Do we have a link to which payment methods we support? If so, it would be great to link anyone looking at the repo to them

Reply:
luckily we have a brand new website for that https://en.komoju.com/payment-methods